### DISCUSSION TITLE
2006/10/08/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-3

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#121] [Announcements]
# 2006/10/08/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-3

昨晚做了个梦，因为特别清晰，所以记录下来。在梦里我指挥着几个弟兄围堵恐怖分子。为了减少伤亡，我发明出了一种罩子，罩在每个人身上，就刀枪不入了。

https://blog.qizhen.xyz/2006/10/08/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-3

**Comment by ruanqizhen** [comment]
之前博客下的留言：

秋
2006年10月16日 08:13 编辑
最近我也经常做奇怪的梦,还有几次居然是哭醒的.但是梦里的故事却像电影里的一样………但是最近却又没有看什么电影或悲剧什么的,故事里的人我始终记不起是什么样子,但是却会让我泣不成声,奇怪……….

回复
Mavis
2006年10月11日 22:10 编辑
莫非是你在围剿我？？

回复
峰.John
2006年10月10日 00:19 编辑
上帝！！！

回复
juicy
2006年10月9日 19:10 编辑
谢谢。

回复
永青
2006年10月9日 09:11 编辑
明显是因为昨天帝国战败了。。。

回复
